migrations/app/20170622113855_region-capacity_view.js

```diff
@@ -0,0 +1,29 @@
+exports.up = function (knex, Promise) {
```

Comment:
This file name should match the view name i.e. `region-capacity_view` => `region_capacity_view`